packages/slate-commons/src/commands/isCursorOnEdgeOfContainer.ts

```diff
@@ -0,0 +1,40 @@
+import { Editor } from 'slate';
+import type { Range, Point } from 'slate';
+import { ReactEditor } from 'slate-react';
+
+export type ContainerEdge = 'top' | 'bottom';
+
+export function isCursorOnEdgeOfContainer(
```

Comment:
I'd like to have shortcuts introduced (for better code readability):

- `isCursorOnFirstLine`
- `isCursorOnLastLine`